fuel_test/check_version.sh

```diff
@@ -7,10 +7,10 @@
 [ $DEBUG == "on" ] && set -x
 
 if [[ ! -d "/tmp/fuel-plugin-xenserver" ]]; then
-	git clone https://review.openstack.org/openstack/fuel-plugin-xenserver /tmp/fuel-plugin-xenserver
+	git clone https://review.openstack.org/openstack/fuel-plugin-xenserver -b ${BRANCH} /tmp/fuel-plugin-xenserver
 fi
 cd /tmp/fuel-plugin-xenserver
-git fetch https://review.openstack.org/openstack/fuel-plugin-xenserver "$FUEL_PLUGIN_REFSPEC"
```

Comment:
I think this fetch is the only place we needed review.openstack.org in the original (pre-depends-on) case?